src/tasks.js

```diff
@@ -19,6 +19,10 @@ let activeTasks = 0
 export async function findAndProcessAQueuedTask() {
   let task
   try {
+    if (activeTasks > 3) {
```

Comment:
Should we be hardcoding this 3 or should we rather have it configurable like the `task_transactions_length`